modules/apps/portal-workflow/portal-workflow-task-web/src/main/java/com/liferay/portal/workflow/task/web/internal/display/context/WorkflowTaskDisplayContext.java

```diff
@@ -685,6 +685,17 @@ public boolean isShowEditURL(WorkflowTask workflowTask) {
 		return showEditURL;
 	}
 
+	public boolean isShowExtraInfo() {
+		if (_showExtraInfo != null) {
+			return _showExtraInfo;
+		}
+
+		_showExtraInfo = ParamUtil.getBoolean(
+			_httpServletRequest, "showExtraInfo", true);
```

Comment:
@ricardocousolr what happens if we don't set the `defaultValue` to `getBoolean`? I mean if we remove the `true` and do `httpServletRequest.setAttribute("showExtraInfo", Boolean.TRUE);` when necessary.

Reply:
@carolmariaabb, could you elaborate on your alternative approach?

Reply:
Hi @ricardocousolr,

After investigating the code a little more, the fix that seems to be the most appropriate would look like this [code](https://github.com/liferay-continuous-integration/liferay-portal/blob/34c002de7efce150a2d69a3749938054a41bec49/modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/result/display/builder/SearchResultContentDisplayBuilder.java#L114-L115), i.e.:

```
public boolean isShowExtraInfo() {
	if (_showExtraInfo != null) {
		return _showExtraInfo;
	}

	if (Objects.equals(
			ParamUtil.getString(_liferayPortletRequest, "type"),
			"document")) {

		_showExtraInfo = true;
	}

	return _showExtraInfo;
}
```
Makes sense to you?

Reply:
Hi @carolmariaabb, 

yes I think targeting documents like that is the most conservative approach. 

I've push a couple of commits to the PR:
- [c3b6ecb](https://github.com/liferay-workflow/liferay-portal/pull/872/commits/c3b6ecb821ab78f6b8ef4bbbe8f81a187427575e) uses your change but making sure that `_showExtraInfo` is set to `fasle` if previously `null` and not of `"document"` type.
- [5e2bd09](https://github.com/liferay-workflow/liferay-portal/pull/872/commits/5e2bd09be15966d63dcbcc83beb0e849cf7280cf) applies the same approach to another display context and view that I hand't noticed existed. These can be accessed from:
1. Personal Menu Icon > My Submissions (`UserWorkflowPortlet`)
2. Upper Menu > Applications > Submissions (`ControlPanelWorkflowInstancePortlet`)

Let me know if you think this is on the right track. Thanks!